src/backend/databaseConfig.go

```
@@ -0,0 +1,313 @@
+    }
+
+	if _, err := c.AddFunc("0 2 * * *", func() {
+		log.Println("Cron job: Running database backup at", time.Now())
+		backupDatabase()
+		cleanupOldBackups()
+	}); err != nil {
+		log.Fatalf("Error scheduling backupDatabase cron job: %v", err)
+	}
+    
+    // scraping wikipedia every 5. minutes
+    c.AddFunc("*/5 * * * *", func() {
```

golangci-lint (1.64.8)

Error return value of `c.AddFunc` is not checked

(errcheck)